packing3d.py

```diff
@@ -241,7 +247,7 @@ def _define_objective(cqm: ConstrainedQuadraticModel, vars: Variables,
 
     # Third term of the objective:
     third_obj_term = quicksum(
-        bins.height * vars.bin_on[j] for j in range(num_bins))
+        bins.height * vars.bin_on[j] for j in range(lowest_num_bin, num_bins))
```

Comment:
I'm curious how changing the range lower bound here works. I know `bin_on` is now fixed to `1` for 1 or more bins, but don't we still need a term in the objective for those bins?